contracts/ZepCore.sol

```diff
@@ -11,23 +11,49 @@ import "zos-core/contracts/upgradeability/UpgradeabilityProxyFactory.sol";
 import "zos-core/contracts/ImplementationProvider.sol";
 import "zeppelin-solidity/contracts/math/SafeMath.sol";
 
+/**
+ * @title ZepCore
+ * @dev This contract controls the kernel distributions and versions for ZeppelinOS
```

Comment:
what about `this contract controls the registry and vouching mechanism of the kernel distributions and versions for ZeppelinOS`?